app/templates/index.html

```diff
@@ -59,7 +59,8 @@
                             <button type="submit" value="bing" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/bing_icon.ico') }}" width="30px" alt="Bing Icon"> Bing</button>
                             <button type="submit" value="duckduckgo" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/duckduckgo_icon.png') }}" width="30px" alt="Duckduckgo Icon"> DuckDuckGo</button>
                             <button type="submit" value="yahoo" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/yahoo_icon.ico') }}" width="30px" alt="Yahoo Icon"> Yahoo</button>
-			    <button type="submit" value="ask" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/ask_icon.ico') }}" width="30px" alt="Ask Icon"> Ask</button>
+			                <button type="submit" value="ask" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/ask_icon.ico') }}" width="30px" alt="Ask Icon"> Ask</button>
```

Comment:
Inconsistent spacing here

Reply:
I am not sure but please see previous PR correctly because these errors are generating from other persons code. I am happy to correct them but this leads to reading of whole code from top to bottom

Reply:
No no, I am just saying that there is an extra tab here